go.mod

```diff
@@ -140,6 +140,7 @@ require (
 	golang.org/x/sys v0.6.0
 	golang.org/x/term v0.6.0
 	golang.org/x/text v0.8.0
+	golang.org/x/time v0.3.0
```

Comment:
I don't see a related import in your changes, so this seems a bit unexpected. Outdated e_imports.go maybe?